Doc/library/sqlite3.rst

```diff
@@ -47,83 +47,104 @@ This document includes four main sections:
       PEP written by Marc-André Lemburg.
 
 
+.. We use the following practises for SQL code:
+.. - UPPERCASE for keywords
+.. - lowercase for schema
+.. - single quotes for string literals
+.. - singular for table names
+.. - if needed, use double quotes for table and column names
+
 .. _sqlite3-tutorial:
 
 Tutorial
 --------
 
-To use the module, start by creating a :class:`Connection` object that
-represents the database.  Here the data will be stored in the
-:file:`example.db` file::
+In this tutorial, you will learn the basics of the :mod:`!sqlite3` API
+by creating an on-disk database :file:`python.db`
+for managing Python release metadata.
+A fundamental understanding of database concepts,
+like `transactions`_ and `cursors`_, is assumed.
+
+To start working with an SQLite database,
+you must first open a connection to it.
+Call the :func:`sqlite3.connect` function
+to create a :class:`Connection` to the database :file:`python.db`::
```

Comment:
```suggestion
to create a connection to the database :file:`python.db`::
```

My original suggestion made a mistake here, IMO—I think its better to use the prose word "connection" here, and then cross-reference the class in the line below where it is explicitly introduced and directly being referred to, rather than having two references a couple of lines apart, one where it isn't really directly called for.

Reply:
+1